osu.Game/Overlays/BeatmapListing/BeatmapSearchScoreFilterRow.cs

```diff
@@ -31,18 +33,36 @@ public RankItem(ScoreRank value)
             {
             }
 
-            protected override string LabelFor(ScoreRank value)
+            protected override LocalisableString LabelFor(ScoreRank value)
```

Comment:
Any reason `ScoreRank` is done this way instead of using a mapper?

Reply:
Nope. I'm not too familiar with this code and didn't have the `LocalisableEnum` attribute at the time. Fixed.